solidity/contracts/examples/Consumer.sol

```diff
@@ -11,16 +12,35 @@ contract Consumer is Chainlinked {
     setOracle(_oracle);
   }
 
-  function requestEthereumPrice(string _currency) public {
-    ChainlinkLib.Run memory run = newRun("someJobId", this, "fulfill(bytes32,bytes32)");
-    run.add("url", "https://etherprice.com/api");
-    string[] memory path = new string[](2);
-    path[0] = "recent";
-    path[1] = _currency;
+  function requestEthereumPrice(string _jobid, string _currency) public {
```

Comment:
How much of a pain would it be to pass `_jobId` into the constructor? Seems like it shouldn't ever change, the rest of the parameters are pretty coupled to it.

Reply:
I actually needed it as a function parameter because I do expect it to change when I want to use my node or our node that we'll be running. Saves time from having to redeploy a new Consumer contract.